rust_src/src/keymap.rs

```diff
@@ -112,4 +113,31 @@ pub fn lookup_key(keymap: LispObject, key: LispObject, accept_default: LispObjec
     }
 }
 
+/// Define COMMAND as a prefix command.  COMMAND should be a symbol.
+/// A new sparse keymap is stored as COMMAND's function definition and its
+/// value.
+/// This prepares COMMAND for use as a prefix key's binding.
+/// If a second optional argument MAPVAR is given, it should be a symbol.
+/// The map is then stored as MAPVAR's value instead of as COMMAND's
+/// value; but COMMAND is still defined as a function.
+/// The third optional argument NAME, if given, supplies a menu name
+/// string for the map.  This is required to use the keymap as a menu.
+/// This function returns COMMAND.
+#[lisp_fn(min = "1")]
+pub fn define_prefix_command(
+    command: LispObject,
+    mapvar: LispObject,
+    name: LispObject,
+) -> LispObject {
+    let map = unsafe { LispObject::from(Fmake_sparse_keymap(name.to_raw())) };
+    unsafe { Fset(command.to_raw(), map.to_raw()) };
```

Comment:
this was a `Ffset` before, and now it's a `Fset`

Reply:
🙈